plugins/plugin-core-graphing/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@syntest/plugin-core-graphing",
+  "version": "0.0.1",
```

Comment:
We should start with a feature version: 0.1.0

Reply:
Fixed